actix/src/sec-5-arbiter.md

```diff
@@ -1 +1,103 @@
-[WIP] 
\ No newline at end of file
+# Arbiter
+
+Arbiters provide an asynchronous execution context for actors. Where an Actor
+contains a Context that defines it's Actor specific execution state, Arbiters
+host the environment where an actor runs.
+
+As a result Arbiters perform a number of function. Most notably, they are able
+to spawn a new OS thread, run an event loop, execute functions asynchronously
+and act as helpers for asynchronous tasks.
+
+## Using Arbiter for resolving async events
+
+If you aren't an expert in rust futures, Arbiter can be a helpful
+and simple wrapper to resolving async events in order. Consider
+we have two actors, A and B, and we want to run an event on B
+only once a result from A is completed. We can use Arbiter::spawn
+to assist with this task.
+
+```rust
+# extern crate actix;
+# extern crate futures;
+# use futures::Future;
+# use actix::prelude::*;
+# 
+# struct SumActor {}
+# 
+# impl Actor for SumActor {
+#     type Context = Context<Self>;
+# }
+
+struct Value(usize, usize);
+
+# impl Message for Value {
+#    type Result = usize;
+# }
+
+impl Handler<Value> for SumActor {
+    type Result = usize;
+
+    fn handle(&mut self, msg: Value, ctx: &mut Context<Self>) -> Self::Result {
+        msg.0 + msg.1
+    }
+}
+
+# struct DisplayActor {}
+# 
+# impl Actor for DisplayActor {
+#     type Context = Context<Self>;
+# }
+
+struct Display(usize);
+
+# impl Message for Display {
+#    type Result = ();
+# }
+
+impl Handler<Display> for DisplayActor {
+    type Result = ();
+
+    fn handle(&mut self, msg: Display, ctx: &mut Context<Self>) -> Self::Result {
+        println!("Got {:?}", msg.0);
+    }
+}
+
+
+fn main() {
+    let system = System::new("test");
```

Comment:
It might be add that `System` actually has own arbiter and therefore in this example you spawn task on System's arbiter

Reply:
Yes, this is a good point.